scripts/python/read_dense.py

```diff
@@ -50,17 +55,40 @@ def read_array(path):
     return np.transpose(array, (1, 0, 2)).squeeze()
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-d", "--depth_map",
+                        help="path to depth map", type=str, required=True)
+    parser.add_argument("-n", "--normal_map",
+                        help="path to normal map", type=str, required=True)
+    parser.add_argument("--min_depth", help="minimum visualization depth",
```

Comment:
I would rename this to min_depth_percentile / max_depth_percentile and also adjust the help description, because these are not absolute depth values.

Reply:
Agreed. What do you think about adding the option for selecting absolute values as well?

Reply:
Absolute values are relatively meaningless here, because the SFM reconstruction has arbitrary scale. I would stick to percentiles.